wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+# Beatmap information
+
+The **beatmap information page** displays various statistics about a [beatmap](/wiki/Beatmap), such as its mapper, star rating, and the BPM. This article details the many aspects and features the information page has to offer.
+
+## Difficulty menu
+
+This menu is the main part containing vital information about the beatmap. The background depicts a cutout version of the beatmap's image.
+
+At the top-left, next to the currently selected `info` tab, is the [beatmap discussion](/wiki/Beatmap_discussion) tab used for [modding](/wiki/Modding) a beatmap.
+
+### Mapping information
+
+::: Infobox
+![](img/mapping-info.png "Mapping information")
+:::
+
+::: Infobox
+![](img/game-mode-selection.png "Game mode selection")
+:::
+
+Directly below the tab selection follows the difficulty section presenting every [difficulty](/wiki/Beatmap/Difficulty) in the beatmap. The game mode icons are coloured based on the [star rating colour spectrum](/wiki/Beatmap/Difficulty#difficulty-and-star-rating).
+
+Next to this display is the difficulty's name, the play count, and the number of users that have added this beatmap to their favourites. Hovering over a different icon than the currently selected one reveals the difficulty's name and the star rating. Clicking the icon will select this difficulty as the active one, resulting in any beatmap statistics on the page to change accordingly. When hovering over the favourite count, up to 50 profile icons of users favouriting this beatmap will be displayed.
+
+Difficulties from other [game modes](/wiki/Game_mode) can be accessed by clicking on the respective icon in the game mode selection. The number next to the icons indicates the number of difficulties created by the beatmap host and any guest mappers. Difficulties mapped in [osu!](/wiki/Game_mode/osu!) are automatically available as [converts](/wiki/Beatmap/Converts) in the other game modes.
+
+To the left side beneath the play count, the title together with the artist is shown. Clicking on the title will automatically search for other beatmaps with the exact same song title. Underneath the artist's name, an icon and the name of the mapper is displayed as well as the upload date and when the beatmap was last changed (e.g. when the beatmap was moved to a certain [category](/wiki/Beatmap/Category)).
+
+Further below these action buttons from left to right may be present, allowing the following:
+
+- **Favourite:** Add this beatmap to the favourites section in the user profile.
```

Comment:
Would explain it in a more direct manner. Sounds too neutral to the reader (what favourites section, whose profile, etc...).
```suggestion
- **Favourite:** Adds this beatmap to the favourites section in your user profile.
```